param/parameterized.py

```diff
@@ -340,7 +340,7 @@ def edit_constant(parameterized):
         p.constant = False
     try:
         yield
-    except:
+    except Exception:
         raise
```

Comment:
Likely this can just be removed or will be removed when running something like pyupgrade